recipes/nest/activate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NEST_INSTALL_DIR=$PREFIX
+NEST_DATA_DIR="$NEST_INSTALL_DIR/share/nest"
+NEST_DOC_DIR="$NEST_INSTALL_DIR/share/doc/nest"
+NEST_MODULE_PATH="$NEST_INSTALL_DIR/lib/nest"
+NEST_PYTHON_PREFIX="$NEST_INSTALL_DIR/lib/python3.6/site-packages"
+PYTHONPATH="$NEST_PYTHON_PREFIX:$PYTHONPATH"
```

Comment:
You shouldn't need to set `PYTHONPATH`, if it is needed the package should be patched instead.
```suggestion
```

Reply:
Thanks, but it es needed

Reply:
Why? The conda provided installation of python looks in there by default.

Reply:
You're absolutely right. I removed it.